File: customer/src/main/java/com/example/customer/auth/RegisterRequest.java
package com.example.customer.auth;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class RegisterRequest {
    public String firstName;
    public String lastName;
    public String email;
    private String username;
    private String password;
}

File: BookingAccomodation/user-service/Model/RegisterRequest.cs
using System.ComponentModel.DataAnnotations;

namespace user_service.Model
{
    public class RegisterRequest
    {
        public string FirstName { get; set; }

        public string LastName { get; set; }

       
        public string Username { get; set; }

        public string Password { get; set; }
    }
}
